src/connection/IConnection.php

```diff
@@ -11,7 +11,10 @@
  */
 interface IConnection
 {
-    public function __construct(string $ip = '127.0.0.1', int $port = 7687, int $timeout = 15);
+    /**
```

Comment:
The annotation here is required? If you want to add it, you should annotate all method arguments.

Reply:
It is because timeout can now be int or float. I'll go ahead and annotate all three arguments